web/themes/joinup/templates/parts/featured--anonymous.html.twig

```diff
@@ -0,0 +1,55 @@
+<div class="section section--featured-lg">
+  <div class="container-fluid">
+    <div class="anonymous-menu">
+      <ul class="anonymous-menu__list">
+        <li class="anonymous-menu__item">
+          <a class="anonymous-menu__link" href="/user/login">Log in</a>
```

Comment:
Do not hardcode paths in twig templates. Use {{ path("user.login") }}.
"Log in" string should be translatable.